### ISSUE TITLE
broken chat

### ISSUE THREAD

**Issue by si-rob** [keybase/client#5631]
Keybase GUI Version: 1.0.18-20170131120020+3f505f9

Chat doesn't appear to be working. tried to initiate a chat convo with someone who's following me / following them, just sits at securing... they don't get a notification or see indication I'm starting a chat. Same goes for them initiating.

**Comment by cjb** [comment]
Thanks for the report, please could you run `keybase log send`?

**Comment by si-rob** [comment]
yep! Log id: 85c725c5b1c109a76b9b071c

**Comment by cjb** [comment]
Ah, I see there's a "globalError" mentioned in the logs -- this would be some text at the top of the window that flashed up on top of a dark background for a few seconds, did you happen to notice what it said?  (Or, can you make it happen again..?)

**Comment by cjb** [comment]
And is your network connection very bouncy?  It looks like the problem might be that you were somehow offline when trying to create the conversation and now it's stuck in that failed state.

If that's it, quitting the app (from the menubar so that the menubar icon disappears) and relaunching the Keybase app would probably fix it..

**Comment by si-rob** [comment]
argh I didn't.
tried to start a chat with @cjb this time it did establish a connection. I also tried chatt from cli which then seemed to show my other chat in the gui.
no reply yet so not sure if it actually did send..
Network is stable. I'll try re-launching the app though

**Comment by mmaybeno** [comment]
Recently I had this issue as well.

Log ID: 14beb15950ff8e4e23082f1c

Trying to send via the cli I get this message:
`ERROR chat message body hash replay detected`

As mentioned above, it could have been related to sending a chat while my vpn was trying to connect...aka bumpy connection. That said, multiple quit/restarts of the application does not resolve the issue. I also tried revoking the key of this device and creating a new one. No luck.

**Comment by mmaybeno** [comment]
1.20 update fixed sending and reading via the cli but only using the command `keybase chat list`

`keybase chat read` still fails

Sent another log, ID: a9648a134b6e02d9fa66291c

**Comment by mmaybeno** [comment]
It seems to be fixed on my end now. @si-rob any luck on your end? I'm still curious why exactly this happened.

**Comment by si-rob** [comment]
@mmaybeno yeah it appears to be working ok for me now. read is also working ok.
